index.html

```diff
@@ -143,6 +143,31 @@ <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials<
             </div>
         </div>
     </section>
+    <section id="blog" class="background-blue pt-5 pb-5 mt-5 mb-5">
+        <div class="container mt-4 mb-4 text-center">
+          <h2 class="mb-5 text-subheading text-lucida text-bold">Blog</h2>
+          <div class="d-inline-flex flex-column flex-md-row mb-5">
+            <a href="#" class="mb-4 position-rel img-container ml-3 mr-3">
+              <img class=" blog-img img-fluid" src="images/blogimg-1.jpg" alt="Blog First Image">
+              <p class="text-left position-abs mb-0 p-4 caption w-100 text-orange font-weight-bold text-proxima">Hypotherapy For Motivation Getting The Drive Back</p>
+            </a>
+            <a href="#" class="mb-4 position-rel img-container ml-3 mr-3">
+              <img class="blog-img img-fluid" src="images/blogimg-2.jpg" alt="Blog Second Image">
+              <p class="text-left position-abs mb-0 p-4 caption w-100 font-weight-bold text-proxima text-orange text-proxima">Learn To Face Difficulty</p>
```

Comment:
it's better to break a long line into multiple so that the reviewer wouldn't have to scroll horizontally to see the changes, as well as for the sake of code readability
`<p class="text-left position-abs mb-0 p-4 caption w-100 font-weight-bold text-proxima text-orange text-proxima">
     Learn To Face Difficulty
</p>`